shell/platform/android/io/flutter/embedding/engine/FlutterJNI.java

```diff
@@ -522,7 +522,10 @@ public void setViewportMetrics(
       int systemGestureInsetTop,
       int systemGestureInsetRight,
       int systemGestureInsetBottom,
-      int systemGestureInsetLeft) {
+      int systemGestureInsetLeft,
+      int[] displayFeaturesBounds,
```

Comment:
Just checking, you seem to be handling bounds as double previously, and int here (and in tests). Is this intended?

Reply:
Yes, int is used in Android and C++ layers, as they represent physical pixels. These get converted to logical pixels in dart. This is true for the screen size and other measurements (lines above this line).

Reply:
Thanks for the explanation!

Reply:
Optional, but it would be nice to make note of this somewhere briefly to make this more readable in the future.